src/app/components/feed/connection-config.factory.js

```diff
@@ -24,7 +24,7 @@
     return function(pluginHandle, wantedInit, jsep, ok) {
       var current = {};
       var requested = null;
-      var wanted = {audio: true, video: true};
+      var wanted = {audio: false, video: true};
```

Comment:
Sure you need to change this IN ADDITION to changing `feed-connection.factory.js`?